src/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.5.1)
+
+project(CMaizeSuperbuild LANGUAGES CXX)
```

Comment:
This project should probably be named "HelloWorld" or something. The superbuild part is really in the top-level lists.

Reply:
I was reading that when doing multiple level CMakeLists you should use the same project name and have it drawn as a prefix build. I can remove it still if you believe that is better for the super-build.

Reply:
I'd be curious to hear their reasoning for this (if they give it), but that sounds like a bad idea to me. The project name shows up in a couple places, notably in the logs. If you use the same name it's harder to distinguish between the driver build (the top-level) and the subbuilds. That's why I vote for different (and more descriptive) names.

Reply:
You're right, I have changed the name till I collect more reasoning as to if it is beneficial or not to have one project name.